src/newusers.c

```diff
@@ -1333,7 +1333,7 @@ int main (int argc, char **argv)
 	sssd_flush_cache (SSSD_DB_PASSWD | SSSD_DB_GROUP);
 
 #ifdef USE_PAM
-	unsigned int i;
+	size_t i;
 	/* Now update the passwords using PAM */
 	for (i = 0; i < nusers; i++) {
```

Comment:
While you're at it, it might be interesting to reduce the scope of `i`, by using a C99 loop variable declaration

Reply:
I like the second patch.

For "src/newusers.c: Turn nusers into size_t":

```
Reviewed-by: Alejandro Colomar <alx@kernel.org>
```

Thanks!

Reply:
Adjusted the scope.